network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -523,12 +523,14 @@ def get_product_subset(cutoff_date, authenticated, key, products, language_code=
     if not authenticated:
         products = products.live()
 
-    products = products.prefetch_related(
-        "evaluation__votes",
-        "image__renditions",
-        "product_categories__category",
+    products = (
+        products.prefetch_related(
+            "image__renditions",
+            "product_categories__category",
+        )
+        .annotate(_average_creepiness=Coalesce(models.Avg("evaluation__votes__value"), float(0)))
```

Comment:
We need to deal with cases where there aren't any votes yet, in which case we want the creepiness to be 0. I tried using the `default` parameter on `models.Avg` but it would return `None` instead. Hence, the `Coalesce`.